packages/api/models/granules.js

```diff
@@ -336,10 +337,10 @@ class Granule extends Manager {
             timeToArchive: get(granule, 'post_to_cmr_duration', 0) / 1000,
             processingStartDateTime: granule.sync_granule_end_time
```

Comment:
In the EMS document table 3.3.1 Ingest Flat file format:
processingStartDateTime | This   is the starting date and time of the ingest process based on GMT. This value   must be earlier than the current date and time and the   processingEndDateTime.  Format:   YYYY-MM-DD HH:MMAMorPM
-- | --
processingEndDateTime | This is the   ending date and time of the ingest process based on GMT.  This value must be earlier than the current   date and time.  Format:   YYYY-MM-DD HH:MMAMorPM

I think the processingStartDateTime is when the ingest granule starts (sync_granule start time or the workflow start time), not when the 'processing' step starts

Reply:
Updated to just use workflow start time and workflow end time following the comments from EMS.